### ISSUE TITLE
load_* functions fail on case sensitive operating systems

### ISSUE THREAD

**Issue by ethanwhite** [mzabrams/fars-cleaner#5]
To reproduce run the following on Linux:

```python
from pathlib import Path
from fars_cleaner import FARSFetcher, load_pipeline

fetcher = FARSFetcher(project_dir=Path("fars-test/"))
vehicles, accidents, people = load_pipeline(fetcher=fetcher, first_run=True, target_folder=Path("fars-tests/"))
```

Result:

```
FileNotFoundError: [Errno 2] No such file or directory: 'fars-test/data/fars/1975.unzip/VEHICLE.csv'
```

This results because the actual file extension is `.CSV` not `.csv`. While these are considered the same on Windows and some installations of macOS, they are not on Linux and other installations of macOS.

While the fix for the extension should be relatively easy there is a larger issue in that the capitalization of the file names changes in 2015. Before 2015 the name before the extension is in all caps. Starting in 2015 the name before the extension is a mix of different cases depending on the file. So this is probably going to require some year specific code or a filename lookup table to address this.

This is a review issue for https://github.com/openjournals/joss-reviews/issues/4678

**Comment by mzabrams** [comment]
Implemented fuzzy-finder matching  to find appropriate files (should be futureproofed as well)

**Comment by mzabrams** [comment]
Closed  with commit 091fa3c26879baf6574a371ba41ccfabcc169b38
